src/components/Staking/StakingStatsBox.tsx

```diff
@@ -190,7 +192,16 @@ const StakingStatsBox: React.FC<IProps> = () => {
         )}
         <Label>
           <Translation id="page-staking-stats-box-metric-2" />
-          <BeaconchainTooltip />
+          <BeaconchainTooltip>
+            <div>
+              <Text>
+                Number of active, non-exited validator accounts on the Beacon
```

Comment:
Doesn't active imply non-exited?

```suggestion
                Number of active validator accounts on the Beacon
```

Reply:
It does to me, but some may interpret "active" as "online"... and inactivated as an offline validator (vs exited). That was my reasoning here... if folks think "active" is enough I'm fine with it